Podfile.lock

```diff
@@ -1,7 +1,7 @@
 PODS:
   - 1PasswordExtension (1.8.5)
   - Alamofire (4.7.3)
-  - Automattic-Tracks-iOS (0.4.1):
+  - Automattic-Tracks-iOS (0.4.2):
```

Comment:
It looks like this PR updated more than just the Authenticator pod based on the changes of the lockfile. With just the two pod changes you can issue:

`bundle exec pod update WordPressAuthenticator WordPressUI`

That way we're not introducing a bunch of other unexpected dependency changes. 😄

Reply:
it's interesting that `pod update WordPressAuthenticator WordPressUI` still updated the `Automattic-Tracks-iOS` pod. Eventually, I had to first set `Automattic-Tracks-iOS` to a fixed version (`pod 'Automattic-Tracks-iOS', '0.4.1'`), run `pod install` and then change it back to `pod 'Automattic-Tracks-iOS', '~> 0.4.0'` to revert the pod version. I was wondering if it's safer to set the pods to a fixed version for the ones we're not actively working on?

Reply:
reverted in: https://github.com/woocommerce/woocommerce-ios/pull/1565/commits/8ffa7ea5204449b18847cc399fe34ff115c29268